cmd/proxy/main.go

```diff
@@ -38,15 +36,9 @@ func main() {
 	if *verbose {
 		log.Printf("Server starting up! - configured to listen on http interface %s and https interface %s", *httpProxyAddr, *tlsProxyAddr)
 	}
-	p := proxy.NewTransparentProxyServer(*verbose)
```

Comment:
i don't love this choice. I think I'd probably lean towards keeping the proxy _server_ construction exposed in the service constructor. mainly i'd be concerned if that constructor had to have more configuration, where would the extra params go. one answer could be a ServerOpts struct which could be added later but I'm not sure that's clearer than or preferable to exposing the server object itself.

Reply:
I think that's a fair point. I left the initialization of the proxy server outside of the service.